src/games/kalambury/data/phrases/index.js

```diff
@@ -0,0 +1,32 @@
+import * as en from "./en";
+import * as pl from "./pl";
+
+const languages = { pl, en };
+
+export const languagesList = [
+  {
+    key: "pl",
+    name: "polski",
+    sets: [
```

Comment:
We're now defining wordlists in two places:
- one is here
- another one is in language-specific indexes
Maybe we could decide on a single place to do this? IMO it would be cleaner to keep this dictionary in language folders - close to actual list files.